src/main/java/org/example/database/SpringConfig.java

```diff
@@ -1,13 +1,18 @@
 package org.example.database;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 
 import javax.sql.DataSource;
 
 @Configuration
+@EnableTransactionManagement
+@PropertySource("classpath:application.properties")
```

Comment:
Why do we need here PropertySource annotation?)

Reply:
i added this line because i used here the annotation to load properties from the application.properties file, so i can easily manage and access configuration values

Reply:
For Spring Boot default property file is application.properties in our src/main/resources directory and it will be auto-detected